test/functional/test_framework/messages.py

```
@@ -2475,6 +2475,40 @@ def __repr__(self):
                (self.protx_hash, self.requested_height, self.quorum_hash)
 
 
+class CSigSharesInv:
+    __slots__ = ("session_id", "inv_size")
```

Ruff (0.15.18)

[warning] `CSigSharesInv.__slots__` is not sorted

Apply a natural sort to `CSigSharesInv.__slots__`

(RUF023)